examples/snmp/pfsense.js

```diff
@@ -0,0 +1,1144 @@
+        exec: snmp_get_exec
+    },
+    {
+        uid: "short_packets",
+        label: "Short packets",
+        type: D.valueType.RATE,
+        unit: "pps",
+        oid: "1.3.6.1.4.1.12325.1.200.1.2.4.0",
+        exec: snmp_get_exec
+    },
+    {
+        uid: "source_tracking_table_current",
+        label: "Source tracking table current",
+        unit: "",
+        oid: "1.3.6.1.4.1.12325.1.200.1.4.1.0",
+        order: 0,
+        exec: snmp_get_exec
+    },
+    {
+        uid: "source_tracking_table_limit",
+        label: "Source tracking table limit",
+        unit: "",
+        oid: "1.3.6.1.4.1.12325.1.200.1.5.2.0",
+        order: 0,
+        exec: snmp_get_exec
+    },
+    {
+        uid: "source_tracking_table_utilization",
+        label: "Source tracking table utilization",
+        unit: "%",
+        exec: [function (last, next) {
+            var count = get_result("1.3.6.1.4.1.12325.1.200.1.4.1.0");
+            var limit = get_result("1.3.6.1.4.1.12325.1.200.1.5.2.0");
+            this.result = (count * 100) / limit;
+            next(this);
+        }, snmp_generate_variable],
+    },
+    {
+        uid: "state_of_nginx_process",
+        walk_root: "1.3.6.1.2.1.25.4.2.1.2",
+        walk_param_name: "nginx",
+        walk_oid: "1.3.6.1.2.1.25.4.2.1.7",
+        exec: snmp_walk_get_exec,
+        label: "State of nginx process",
+        unit: "",
+    },
+    {
+        uid: "states_table_current",
+        label: "States table current",
+        unit: "",
+        oid: "1.3.6.1.4.1.12325.1.200.1.3.1.0",
+        order: 0,
+        exec: snmp_get_exec
+    },
+    {
+        uid: "states_table_limit",
+        label: "States table limit",
+        unit: "",
+        oid: "1.3.6.1.4.1.12325.1.200.1.5.1.0",
+        order: 0,
+        exec: snmp_get_exec
+    },
+    {
+        uid: "states_table_utilization",
+        label: "States table utilization",
+        unit: "%",
+        exec: [function (last, next) {
+            var count = get_result("1.3.6.1.4.1.12325.1.200.1.3.1.0");
+            var limit = get_result("1.3.6.1.4.1.12325.1.200.1.5.1.0");
+            this.result = (count * 100) / limit;
+            next(this);
+        }, snmp_generate_variable],
+    },
+    {
+        uid: "system_contact_details",
+        label: "System contact details",
+        unit: "",
+        oid: "1.3.6.1.2.1.1.4.0",
+        exec: snmp_get_exec
+    },
+    {
+        uid: "system_description",
+        label: "System description",
+        unit: "",
+        oid: "1.3.6.1.2.1.1.1.0",
+        exec: snmp_get_exec
+    },
+    {
+        uid: "system_location",
+        label: "System location",
+        unit: "",
+        oid: "1.3.6.1.2.1.1.6.0",
+        exec: snmp_get_exec
+    },
+    {
+        uid: "system_name",
+        label: "System name",
+        unit: "",
+        oid: "1.3.6.1.2.1.1.5.0",
+        exec: snmp_get_exec
+    },
+    {
+        uid: "system_object_id",
```

Comment:
Not particularly useful in this driver (domotz is already fetching SNMP information from the 1.3.6.1.2.1.1)

Please remove this variable